src/CoherenceBuild/PackagePublisher.cs

```diff
@@ -51,6 +53,15 @@ public static void PublishToFeed(ProcessResult processResult, string feed, strin
                  processResult.CoreCLRPackages.Values,
                  processResult.ProductPackages.OrderBy(p => p.Value.Degree).Select(p => p.Value));
 
+            var httpClient = new System.Net.Http.HttpClient();
+            var indexJson = JObject.Parse(httpClient.GetAsync(feed.TrimEnd('/') + "/api/v3/index.json").Result.Content.ReadAsStringAsync().Result);
+            var v3Feed = indexJson
+                .Property("resources")
+                .Value.AsJEnumerable().Cast<JObject>()
+                .First(item => item.Property("@type").Value.ToString() == "PackageBaseAddress/3.0.0")
+                .Property("@id")
+                .Value.ToString();
```

Comment:
I think there's a chance to get some nulls while parsing this. Maybe use the null conditional to avoid that?